src/main/cpp/t09_spiral.cpp

```diff
@@ -21,7 +21,71 @@
 
 using namespace std;
 
-int t09_spiral() {
+int t09_spiral() { 
+    int n, m, k = 1, l = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'k' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4488444)